label_studio/webhooks/api.py

```diff
@@ -0,0 +1,125 @@
+import logging
+
+
+from django.utils.decorators import method_decorator
+from django_filters.rest_framework import DjangoFilterBackend
+import django_filters
+from drf_yasg.utils import swagger_auto_schema
+from drf_yasg import openapi
+from rest_framework import generics
+from rest_framework.decorators import action
+from rest_framework.permissions import AllowAny, IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .models import Webhook, WebhookAction
+from .serializers import WebhookSerializer, WebhookSerializerForUpdate
+from projects import models as project_models
+
+
+class WebhookFilterSet(django_filters.FilterSet):
+    project = django_filters.ModelChoiceFilter(
+        field_name='project', queryset=project_models.Project.objects.all(), null_label='isnull'
+    )
+
+
+@method_decorator(
+    name='get',
+    decorator=swagger_auto_schema(
+        tags=['Webhooks'],
+        operation_summary='List of webhooks',
+        operation_description="List of webhooks of user's active organization.",
+    ),
+)
+@method_decorator(
+    name='post',
+    decorator=swagger_auto_schema(
+        tags=['Webhooks'],
+        operation_summary='Create a webhook',
+        operation_description="Create a webhook for user's active organization.",
```

Comment:
```suggestion
        operation_description="Create a webhook for your organization.",
```

Reply:
Might also need to add a reference to the serializer here if it doesn't show up in the API docs, but only if there is content required for the request body, like so:

```suggestion
        operation_description="Create a webhook for user's active organization.",
        request_body=WebhookSerializer
```

Not sure tho